tools/accuracy_checker/README.md

```diff
@@ -204,6 +204,8 @@ example of dataset definition:
     - type: accuracy
 ```
 
+If you want to evaluate models using prepared config files and well-known datasets, you need to organize folders with validation datasets in a certain way. More detailed information about datasets preparation you can find in [Dataset Preparation Guide](./datasets_preparation.md).
```

Comment:
logically it should be higher, after:
`
If your dataset data is a well-known competition problem (COCO, Pascal VOC, ...) and/or can be potentially reused for other models it is reasonable to declare it in some global configuration file (definition file). This way in your local configuration file you can provide only name and all required steps will be picked from global one. To pass path to this global configuration use --definition argument of CLI.
`

Reply:
Done